lomap/graphgen.py

```diff
@@ -726,21 +760,22 @@ def max_dist_mol(mol):
                 max_dist = max_dist_mol(mol)
 
                 if max_dist < self.max_mol_size:
-                    
-                    mol = AllChem.RemoveHs(mol)
-        
-                    AllChem.Compute2DCoords(mol)
-                
                     fname = os.path.join(directory_name, self.dbase[id_mol].getName() + ".png")
-                    
-                    Draw.MolToFile(mol, fname, size=(100,100), fitimage=True, imageType='png' )
-                    
+                    #sliu 07/17 
+                    #1, modify here to calculate the 2D structure for ligands cannot remove Hs by rdkit
+                    #2, change the graph size to get better resolution            
+                    try:
+                        mol = AllChem.RemoveHs(mol)
+                        AllChem.Compute2DCoords(mol)
+                        Draw.MolToFile(mol, fname, size=(200,200), kekulize=False, fitimage=True, imageType='png' )
+                    except:
+                        logging.info("This molecule %s has trouble to remove Hs using rdkit"%self.dbase[id_mol].getName())
```

Comment:
OK. So we should:
a) document this in the code
b) Give the RDKit people some example molecules and ask them to try to fix the issue
c) Document in the code that we've done so

Then we can revisit this part of the code once the RDKit folks fix it.